async-nats/tests/object_store.rs

```diff
@@ -110,6 +114,111 @@ mod object_store {
+            .unwrap();
+
+        bucket
+            .put("FOO", &mut io::Cursor::new(vec![2, 3, 4, 5]))
+            .await
+            .unwrap();
+
+        let original_info = bucket.info("FOO").await.unwrap();
+
+        tokio::time::sleep(Duration::from_secs(1)).await;
+
+        let new_info = bucket.info("FOO").await.unwrap();
+
+        // We should get back the same modified datetime
+        assert_eq!(original_info.modified, new_info.modified);
+
+        bucket
+            .put("FOO", &mut io::Cursor::new(vec![2, 3, 4, 5]))
+            .await
+            .unwrap();
+
+        let new_info = bucket.info("FOO").await.unwrap();
+
+        // We should get back different modified datetime because we've modified
+        assert_ne!(original_info.modified, new_info.modified);
+    }
+
+    #[tokio::test]
+    #[ignore]
+    async fn object_info_modified_with_cli() {
+        let server = nats_server::run_server("tests/configs/jetstream.conf");
+
+        let client = async_nats::connect(server.client_url()).await.unwrap();
+        let jetstream = async_nats::jetstream::new(client);
+
+        let bucket = jetstream
+            .create_object_store(async_nats::jetstream::object_store::Config {
+                bucket: "bucket".to_string(),
+                ..Default::default()
+            })
+            .await
+            .unwrap();
+
+        // Work with nats cli object store to create an object
+        // Create a temp file to be used for object store with cli
+        // TODO: Ensure that this works in CI
+        let temp_folder = std::env::temp_dir();
+        let temp_folder_path = std::path::Path::new(&temp_folder);
+
+        let file_path = temp_folder_path.join("FOO.txt");
+        let mut file = File::create(file_path.clone()).await.unwrap();
+        file.write_all(b"test").await.unwrap();
+        file.flush().await.unwrap();
+        drop(file);
+
+        // Shell out to cli -> create an object with test file
+        let mut nats_cmd = tokio::process::Command::new("nats");
+        let cmd_output = nats_cmd
+            .arg("--server")
+            .arg(server.client_url())
+            .arg("object")
+            .arg("put")
+            .arg("bucket")
+            .arg(file_path.clone().to_str().unwrap())
+            .arg("--force")
+            .output()
+            .await
+            .unwrap();
+
+        dbg!(cmd_output);
+
+        // Remove the temp file, we don't need it anymore
+        tokio::fs::remove_file(file_path.clone()).await.unwrap();
+
+        let info = bucket.info(file_path.to_str().unwrap()).await.unwrap();
+        let obj_info = info.clone();
+        dbg!(obj_info);
+
+        let current_time = OffsetDateTime::now_utc();
+        let diff = current_time - info.modified;
+
+        println!("duration diff: {diff}", diff = diff.as_seconds_f64());
+        println!("mtime: {mtime}", mtime = info.modified);
+
+        // TODO: Assert that the mtime diff is within an acceptable threshold
```

Comment:
Note that there is no `assert` here, because I've observed some weird behaviour when creating an object through the nats cli. Namely, when you create an object (`put`) using the CLI, and then query it back using the `async_nats` client, the modified time (`mtime`) is returned as `0001-01-01 0:00:00.0 +00:00:00`, where if you do a `put` using the `async_nats` client and then query it (even if you restart the server), you get back the modified time correctly. When you query the object store with the CLI the results are correct (no matter where the object was created from). 

This is an object created by the CLI and requested by the `async_nats` client (watch for the `modified` field):
```
[async-nats/tests/object_store.rs:202] obj_info = ObjectInfo {
    name: "FOO.txt",
    description: None,
    link: None,
    bucket: "bucket",
    nuid: "Ejl3uiU8wB24SC6fCBo2T7",
    size: 4,
    chunks: 1,
    modified: 0001-01-01 0:00:00.0 +00:00:00, 
    digest: Some(
        "SHA-256=n4bQgYhMfWWaL-qgxVrQFaO_TxsrC4Is0V1sFbDwCgg=",
    ),
    deleted: false,
}
```


This is an object created by the `async_nats` client and then requested by the same: 

```
[async-nats/tests/object_store.rs:207] obj_info = ObjectInfo {
    name: "FOO.txt",
    description: None,
    link: None,
    bucket: "bucket",
    nuid: "RkHZfBa7tqS6UcTWiUwMbv",
    size: 4,
    chunks: 1,
    modified: 2023-04-14 20:16:09.976078 +00:00:00, 
    digest: Some(
        "SHA-256=Db6mTUJ1Yny70JLnKkTr_jDNwXCZO1OcxbpYfOMJldQ=",
    ),
    deleted: false,
}
```

I've attempted to run `nats-server` with `--debug` and `--trace` and watch for the payloads. This is the output for each scenario:

PUT with CLI
--
```[8218] 2023/04/17 16:15:22.253828 [DBG] 127.0.0.1:49487 - cid:8 - Client connection created
[8218] 2023/04/17 16:15:22.254619 [TRC] 127.0.0.1:49487 - cid:8 - <<- [CONNECT {"verbose":false,"pedantic":false,"tls_required":false,"name":"NATS CLI Version 0.0.35","lang":"go","version":"1.19.0","protocol":1,"echo":true,"headers":true,"no_responders":true}]
[8218] 2023/04/17 16:15:22.254794 [TRC] 127.0.0.1:49487 - cid:8 - <<- [PING]
[8218] 2023/04/17 16:15:22.254811 [TRC] 127.0.0.1:49487 - cid:8 - ->> [PONG]
[8218] 2023/04/17 16:15:22.255066 [TRC] 127.0.0.1:49487 - cid:8 - <<- [SUB _INBOX.BdRBVLdFQojDJHRJgYgqAG.*  1]
[8218] 2023/04/17 16:15:22.255115 [TRC] 127.0.0.1:49487 - cid:8 - <<- [PUB $JS.API.STREAM.INFO.OBJ_bucket _INBOX.BdRBVLdFQojDJHRJgYgqAG.7A1xvX2D 0]
[8218] 2023/04/17 16:15:22.255130 [TRC] 127.0.0.1:49487 - cid:8 - <<- MSG_PAYLOAD: [""]
[8218] 2023/04/17 16:15:22.256058 [TRC] 127.0.0.1:49487 - cid:8 - ->> [MSG _INBOX.BdRBVLdFQojDJHRJgYgqAG.7A1xvX2D 1 820]
[8218] 2023/04/17 16:15:22.256094 [TRC] ACCOUNT - <<- [PUB $JS.EVENT.ADVISORY.API  1268]
[8218] 2023/04/17 16:15:22.256167 [TRC] ACCOUNT - <<- MSG_PAYLOAD: ["{\"type\":\"io.nats.jetstream.advisory.v1.api_audit\",\"id\":\"jRaLURzReTbSC87E3jZHpb\",\"timestamp\":\"2023-04-17T13:15:22.255956Z\",\"server\":\"NCJOMMYLGN4EDYH7OOUUQLRF4ZUNKKJZSWMZYPJ5FUXVER323LSLF5JP\",\"client\":{\"acc\":\"$G\",\"rtt\":940000,\"server\":\"NCJOMMYLGN4EDYH7OOUUQLRF4ZUNKKJZSWMZYPJ5FUXVER323LSLF5JP\"},\"subject\":\"$JS.API.STREAM.INFO.OBJ_bucket\",\"response\":\"{\\\"type\\\":\\\"io.nats.jetstream.api.v1.stream_info_response\\\",\\\"total\\\":0,\\\"offset\\\":0,\\\"limit\\\":0,\\\"config\\\":{\\\"name\\\":\\\"OBJ_bucket\\\",\\\"subjects\\\":[\\\"$O.bucket.C.\\\\u003e\\\",\\\"$O.bucket.M.\\\\u003e\\\"],\\\"retention\\\":\\\"limits\\\",\\\"max_consumers\\\":-1,\\\"max_msgs\\\":-1,\\\"max_bytes\\\":-1,\\\"max_age\\\":0,\\\"max_msgs_per_subject\\\":-1,\\\"max_msg_size\\\":-1,\\\"discard\\\":\\\"new\\\",\\\"storage\\\":\\\"file\\\",\\\"num_replicas\\\":1,\\\"duplicate_window\\\":120000000000,\\\"allow_direct\\\":false,\\\"mirror_direct\\\":false,\\\"sealed\\\":false,\\\"deny_delete\\\":false,\\\"deny_purge\\\":false,\\\"allow_rollup_hdrs\\\":true},\\\"created\\\":\\\"2023-04-17T12:43:25.000639Z\\\",\\\"state\\\":{\\\"messages\\\":8,\\\"bytes\\\":14394,\\\"first_seq\\\":3,\\\"first_ts\\\":\\\"2023-04-17T12:44:17.546074Z\\\",\\\"last_seq\\\":18,\\\"last_ts\\\":\\\"2023-04-17T13:13:59.238718Z\\\",\\\"num_subjects\\\":8,\\\"num_deleted\\\":8,\\\"consumer_count\\\":0},\\\"cluster\\\":{\\\"leader\\\":\\\"NCJOMMYLGN4EDYH7OOUUQLRF4ZUNKKJZSWMZYPJ5FUXVER323LSLF5JP\\\"}}\"}"]
[8218] 2023/04/17 16:15:22.256532 [TRC] 127.0.0.1:49487 - cid:8 - <<- [PUB $JS.API.STREAM.MSG.GET.OBJ_bucket _INBOX.BdRBVLdFQojDJHRJgYgqAG.cKDqk4dj 39]
[8218] 2023/04/17 16:15:22.256550 [TRC] 127.0.0.1:49487 - cid:8 - <<- MSG_PAYLOAD: ["{\"last_by_subj\":\"$O.bucket.M.VEVTVA==\"}"]
[8218] 2023/04/17 16:15:22.257225 [TRC] 127.0.0.1:49487 - cid:8 - ->> [MSG _INBOX.BdRBVLdFQojDJHRJgYgqAG.cKDqk4dj 1 502]
[8218] 2023/04/17 16:15:22.257564 [TRC] 127.0.0.1:49487 - cid:8 - <<- [PUB $JS.API.STREAM.MSG.GET.OBJ_bucket _INBOX.BdRBVLdFQojDJHRJgYgqAG.M7PiPjHE 39]
[8218] 2023/04/17 16:15:22.257586 [TRC] 127.0.0.1:49487 - cid:8 - <<- MSG_PAYLOAD: ["{\"last_by_subj\":\"$O.bucket.M.VEVTVA==\"}"]
[8218] 2023/04/17 16:15:22.257697 [TRC] 127.0.0.1:49487 - cid:8 - ->> [MSG _INBOX.BdRBVLdFQojDJHRJgYgqAG.M7PiPjHE 1 502]
[8218] 2023/04/17 16:15:22.257900 [TRC] 127.0.0.1:49487 - cid:8 - <<- [SUB _INBOX.44Ngyg.*  2]
[8218] 2023/04/17 16:15:22.257921 [TRC] 127.0.0.1:49487 - cid:8 - <<- [PUB $O.bucket.C.BdRBVLdFQojDJHRJgYgqFJ _INBOX.44Ngyg.OEwZYQ 683]
[8218] 2023/04/17 16:15:22.257980 [TRC] 127.0.0.1:49487 - cid:8 - <<- MSG_PAYLOAD: ["# Contributing\n\nHello there and welcome!\n\nWhen contributing to this repository, please first discuss the change you wish to make via issue before making a change to avoid duplicate or un-warranted work.\n\n## Pull Request Process\n\n- We use linear history, please make sure your commits are atomic.\n- Do your best to keep commits to a reasonable size, split out into a different pull request for tangental changes that are observable externally.\n- Write well formed commit messages (e.g [Example](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html))\n- Do call out if you start working on an issue and raise pull requests early for visibility to avoid duplicate work.\n"]
[8218] 2023/04/17 16:15:22.260445 [TRC] 127.0.0.1:49487 - cid:8 - <<- [HPUB $O.bucket.M.VEVTVA== _INBOX.44Ngyg.H47WcN 30 248]
[8218] 2023/04/17 16:15:22.260479 [TRC] 127.0.0.1:49487 - cid:8 - <<- MSG_PAYLOAD: ["NATS/1.0\r\nNats-Rollup: sub\r\n\r\n{\"name\":\"TEST\",\"options\":{\"max_chunk_size\":131072},\"bucket\":\"bucket\",\"nuid\":\"BdRBVLdFQojDJHRJgYgqFJ\",\"size\":683,\"mtime\":\"0001-01-01T00:00:00Z\",\"chunks\":1,\"digest\":\"SHA-256=2tfiLxdjp08aB8twSYgv2x1RJg46ZpY4lSlcoTvlJFY=\"}"]
[8218] 2023/04/17 16:15:22.260540 [TRC] 127.0.0.1:49487 - cid:8 - ->> [MSG _INBOX.44Ngyg.OEwZYQ 2  33]
[8218] 2023/04/17 16:15:22.310472 [TRC] 127.0.0.1:49487 - cid:8 - ->> [MSG _INBOX.44Ngyg.H47WcN 2  33]
[8218] 2023/04/17 16:15:22.310598 [TRC] 127.0.0.1:49487 - cid:8 - <<- [PUB $JS.API.STREAM.PURGE.OBJ_bucket _INBOX.BdRBVLdFQojDJHRJgYgqAG.nV2gjrRL 47]
[8218] 2023/04/17 16:15:22.310606 [TRC] 127.0.0.1:49487 - cid:8 - <<- MSG_PAYLOAD: ["{\"filter\":\"$O.bucket.C.Wj5qG9mltM7pjAa4Wh5t2D\"}"]
[8218] 2023/04/17 16:15:22.310716 [TRC] 127.0.0.1:49487 - cid:8 - ->> [MSG _INBOX.BdRBVLdFQojDJHRJgYgqAG.nV2gjrRL 1 83]
[8218] 2023/04/17 16:15:22.310724 [TRC] ACCOUNT - <<- [PUB $JS.EVENT.ADVISORY.API  506]
[8218] 2023/04/17 16:15:22.310737 [TRC] ACCOUNT - <<- MSG_PAYLOAD: ["{\"type\":\"io.nats.jetstream.advisory.v1.api_audit\",\"id\":\"jRaLURzReTbSC87E3jZHqI\",\"timestamp\":\"2023-04-17T13:15:22.310695Z\",\"server\":\"NCJOMMYLGN4EDYH7OOUUQLRF4ZUNKKJZSWMZYPJ5FUXVER323LSLF5JP\",\"client\":{\"acc\":\"$G\",\"rtt\":940000,\"server\":\"NCJOMMYLGN4EDYH7OOUUQLRF4ZUNKKJZSWMZYPJ5FUXVER323LSLF5JP\"},\"subject\":\"$JS.API.STREAM.PURGE.OBJ_bucket\",\"request\":\"{\\\"filter\\\":\\\"$O.bucket.C.Wj5qG9mltM7pjAa4Wh5t2D\\\"}\",\"response\":\"{\\\"type\\\":\\\"io.nats.jetstream.api.v1.stream_purge_response\\\",\\\"success\\\":true,\\\"purged\\\":1}\"}"]
[8218] 2023/04/17 16:15:22.311293 [DBG] 127.0.0.1:49487 - cid:8 - Client connection closed: Client Closed
[8218] 2023/04/17 16:15:22.311318 [TRC] 127.0.0.1:49487 - cid:8 - <-> [DELSUB 1]
[8218] 2023/04/17 16:15:22.311326 [TRC] 127.0.0.1:49487 - cid:8 - <-> [DELSUB 2]
```

PUT with `async_nats`
--
```
[9267] 2023/04/17 16:25:29.935608 [DBG] 127.0.0.1:49511 - cid:8 - Client connection created
[9267] 2023/04/17 16:25:29.936888 [TRC] 127.0.0.1:49511 - cid:8 - <<- [CONNECT {"verbose":false,"pedantic":false,"jwt":null,"nkey":null,"sig":null,"name":null,"echo":true,"lang":"rust","version":"0.29.0","protocol":1,"tls_required":false,"user":null,"pass":[REDACTED],"auth_token":null,"headers":true,"no_responders":true}]
[9267] 2023/04/17 16:25:29.936983 [TRC] 127.0.0.1:49511 - cid:8 - <<- [PING]
[9267] 2023/04/17 16:25:29.936990 [TRC] 127.0.0.1:49511 - cid:8 - ->> [PONG]
[9267] 2023/04/17 16:25:29.940071 [TRC] 127.0.0.1:49511 - cid:8 - <<- [SUB _INBOX.BOhgplvn6AKrHqQNYpLQSB 0]
[9267] 2023/04/17 16:25:29.940098 [TRC] 127.0.0.1:49511 - cid:8 - <<- [PUB $JS.API.STREAM.INFO.OBJ_bucket _INBOX.BOhgplvn6AKrHqQNYpLQSB 4]
[9267] 2023/04/17 16:25:29.940107 [TRC] 127.0.0.1:49511 - cid:8 - <<- MSG_PAYLOAD: ["null"]
[9267] 2023/04/17 16:25:29.940545 [TRC] 127.0.0.1:49511 - cid:8 - ->> [MSG _INBOX.BOhgplvn6AKrHqQNYpLQSB 0 821]
[9267] 2023/04/17 16:25:29.941155 [TRC] ACCOUNT - <<- [PUB $JS.EVENT.ADVISORY.API  1287]
[9267] 2023/04/17 16:25:29.941157 [TRC] 127.0.0.1:49511 - cid:8 - <<- [PING]
[9267] 2023/04/17 16:25:29.941205 [TRC] 127.0.0.1:49511 - cid:8 - ->> [PONG]
[9267] 2023/04/17 16:25:29.941213 [TRC] ACCOUNT - <<- MSG_PAYLOAD: ["{\"type\":\"io.nats.jetstream.advisory.v1.api_audit\",\"id\":\"0tyy4QDMxkK56870UWwR8I\",\"timestamp\":\"2023-04-17T13:25:29.940496Z\",\"server\":\"NALXAP7WEQDE6VGMJPHXLDOAFYBSUUUXWZKYBME5FKNMYLMHRD3YD7V2\",\"client\":{\"acc\":\"$G\",\"rtt\":1396000,\"server\":\"NALXAP7WEQDE6VGMJPHXLDOAFYBSUUUXWZKYBME5FKNMYLMHRD3YD7V2\"},\"subject\":\"$JS.API.STREAM.INFO.OBJ_bucket\",\"request\":\"null\",\"response\":\"{\\\"type\\\":\\\"io.nats.jetstream.api.v1.stream_info_response\\\",\\\"total\\\":0,\\\"offset\\\":0,\\\"limit\\\":0,\\\"config\\\":{\\\"name\\\":\\\"OBJ_bucket\\\",\\\"subjects\\\":[\\\"$O.bucket.C.\\\\u003e\\\",\\\"$O.bucket.M.\\\\u003e\\\"],\\\"retention\\\":\\\"limits\\\",\\\"max_consumers\\\":-1,\\\"max_msgs\\\":-1,\\\"max_bytes\\\":-1,\\\"max_age\\\":0,\\\"max_msgs_per_subject\\\":-1,\\\"max_msg_size\\\":-1,\\\"discard\\\":\\\"new\\\",\\\"storage\\\":\\\"file\\\",\\\"num_replicas\\\":1,\\\"duplicate_window\\\":120000000000,\\\"allow_direct\\\":false,\\\"mirror_direct\\\":false,\\\"sealed\\\":false,\\\"deny_delete\\\":false,\\\"deny_purge\\\":false,\\\"allow_rollup_hdrs\\\":true},\\\"created\\\":\\\"2023-04-17T12:43:25.000639Z\\\",\\\"state\\\":{\\\"messages\\\":8,\\\"bytes\\\":14394,\\\"first_seq\\\":3,\\\"first_ts\\\":\\\"2023-04-17T12:44:17.546074Z\\\",\\\"last_seq\\\":22,\\\"last_ts\\\":\\\"2023-04-17T13:23:03.078012Z\\\",\\\"num_subjects\\\":8,\\\"num_deleted\\\":12,\\\"consumer_count\\\":0},\\\"cluster\\\":{\\\"leader\\\":\\\"NALXAP7WEQDE6VGMJPHXLDOAFYBSUUUXWZKYBME5FKNMYLMHRD3YD7V2\\\"}}\"}"]
[9267] 2023/04/17 16:25:29.941782 [TRC] 127.0.0.1:49511 - cid:8 - <<- [SUB _INBOX.BOhgplvn6AKrHqQNYpLQUE 1]
[9267] 2023/04/17 16:25:29.941796 [TRC] 127.0.0.1:49511 - cid:8 - <<- [PUB $JS.API.STREAM.MSG.GET.OBJ_bucket _INBOX.BOhgplvn6AKrHqQNYpLQUE 43]
[9267] 2023/04/17 16:25:29.941802 [TRC] 127.0.0.1:49511 - cid:8 - <<- MSG_PAYLOAD: ["{\"last_by_subj\":\"$O.bucket.M.VkVWVFZBPT0=\"}"]
[9267] 2023/04/17 16:25:29.941918 [TRC] 127.0.0.1:49511 - cid:8 - ->> [MSG _INBOX.BOhgplvn6AKrHqQNYpLQUE 1 130]
[9267] 2023/04/17 16:25:29.941926 [TRC] ACCOUNT - <<- [PUB $JS.EVENT.ADVISORY.API  558]
[9267] 2023/04/17 16:25:29.941950 [TRC] ACCOUNT - <<- MSG_PAYLOAD: ["{\"type\":\"io.nats.jetstream.advisory.v1.api_audit\",\"id\":\"0tyy4QDMxkK56870UWwRAK\",\"timestamp\":\"2023-04-17T13:25:29.941894Z\",\"server\":\"NALXAP7WEQDE6VGMJPHXLDOAFYBSUUUXWZKYBME5FKNMYLMHRD3YD7V2\",\"client\":{\"acc\":\"$G\",\"rtt\":1396000,\"server\":\"NALXAP7WEQDE6VGMJPHXLDOAFYBSUUUXWZKYBME5FKNMYLMHRD3YD7V2\"},\"subject\":\"$JS.API.STREAM.MSG.GET.OBJ_bucket\",\"request\":\"{\\\"last_by_subj\\\":\\\"$O.bucket.M.VkVWVFZBPT0=\\\"}\",\"response\":\"{\\\"type\\\":\\\"io.nats.jetstream.api.v1.stream_msg_get_response\\\",\\\"error\\\":{\\\"code\\\":404,\\\"err_code\\\":10037,\\\"description\\\":\\\"no message found\\\"}}\"}"]
[9267] 2023/04/17 16:25:29.942241 [TRC] 127.0.0.1:49511 - cid:8 - <<- [UNSUB 0]
[9267] 2023/04/17 16:25:29.953473 [TRC] 127.0.0.1:49511 - cid:8 - <-> [DELSUB 0]
[9267] 2023/04/17 16:25:29.953488 [TRC] 127.0.0.1:49511 - cid:8 - <<- [SUB _INBOX.BOhgplvn6AKrHqQNYpLQYK 2]
[9267] 2023/04/17 16:25:29.953499 [TRC] 127.0.0.1:49511 - cid:8 - <<- [PUB $O.bucket.C.BOhgplvn6AKrHqQNYpLQWH _INBOX.BOhgplvn6AKrHqQNYpLQYK 4]
[9267] 2023/04/17 16:25:29.953508 [TRC] 127.0.0.1:49511 - cid:8 - <<- MSG_PAYLOAD: ["\x02\x03\x04\x05"]
[9267] 2023/04/17 16:25:29.954067 [TRC] 127.0.0.1:49511 - cid:8 - <<- [UNSUB 1]
[9267] 2023/04/17 16:25:29.954105 [TRC] 127.0.0.1:49511 - cid:8 - ->> [MSG _INBOX.BOhgplvn6AKrHqQNYpLQYK 2  33]
[9267] 2023/04/17 16:25:29.985147 [TRC] 127.0.0.1:49511 - cid:8 - <-> [DELSUB 1]
[9267] 2023/04/17 16:25:29.985917 [TRC] 127.0.0.1:49511 - cid:8 - <<- [SUB _INBOX.BOhgplvn6AKrHqQNYpLQaN 3]
[9267] 2023/04/17 16:25:29.985933 [TRC] 127.0.0.1:49511 - cid:8 - <<- [HPUB $O.bucket.M.VEVTVA== _INBOX.BOhgplvn6AKrHqQNYpLQaN 30 248]
[9267] 2023/04/17 16:25:29.985956 [TRC] 127.0.0.1:49511 - cid:8 - <<- MSG_PAYLOAD: ["NATS/1.0\r\nNats-Rollup: sub\r\n\r\n{\"name\":\"TEST\",\"description\":null,\"link\":null,\"bucket\":\"bucket\",\"nuid\":\"BOhgplvn6AKrHqQNYpLQWH\",\"size\":4,\"chunks\":1,\"mtime\":\"2023-04-17T13:25:29.985649Z\",\"digest\":\"SHA-256=Db6mTUJ1Yny70JLnKkTr_jDNwXCZO1OcxbpYfOMJldQ=\"}"]
[9267] 2023/04/17 16:25:29.986106 [TRC] 127.0.0.1:49511 - cid:8 - ->> [MSG _INBOX.BOhgplvn6AKrHqQNYpLQaN 3  33]
[9267] 2023/04/17 16:25:29.986448 [DBG] 127.0.0.1:49511 - cid:8 - Client connection closed: Client Closed
[9267] 2023/04/17 16:25:29.997357 [TRC] 127.0.0.1:49511 - cid:8 - <-> [DELSUB 2]
[9267] 2023/04/17 16:25:29.997364 [TRC] 127.0.0.1:49511 - cid:8 - <-> [DELSUB 3]
```
The only difference that I can spot is that `mtime` is not set with the cli, where it's set with the `async_nats` client. I assume that `mtime` should ideally be set by the server.